pkg/cmd/roachtest/roachtestutil/mixedversion/steps.go

```diff
@@ -814,31 +855,26 @@ func (s panicNodeStep) Run(ctx context.Context, l *logger.Logger, rng *rand.Rand
 	)
 	customStartOpts := restartSystemSettings(true, s.initTarget)
 
-	h.runner.monitor.ExpectProcessDead(s.targetNode)
-
-	const stmt = "SELECT crdb_internal.force_panic('expected panic from panicNodeMutator')"
-	err = h.System.ExecWithGateway(
-		rng,
-		s.targetNode,
-		stmt,
-	)
-
-	if err == nil {
-		return errors.Errorf("expected panic statement to fail, but it succeeded on %s", s.targetNode)
-	}
-
 	startCtx, cancel := context.WithTimeout(ctx, startTimeout)
 	defer cancel()
 
-	return h.runner.cluster.StartE(
+	err = h.runner.cluster.StartE(
 		startCtx,
 		l,
 		startOpts(customStartOpts...),
 		settings,
 		s.targetNode,
 	)
+	if err != nil {
+		return errors.Wrapf(
+			err, "failed to start node %d with binary %s", s.targetNode[0], binary,
+		)
+	}
+	h.runner.monitor.ExpectProcessAlive(s.targetNode)
```

Comment:
`StartE` should already set `s.targetNode` to alive again. Were you experiencing otherwise?